runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -411,22 +411,24 @@ task examplesJavaFnApiWorkerIntegrationTest(type: Test) {
 }
 
 task coreSDKJavaLegacyWorkerIntegrationTest(type: Test) {
-  group = "Verification"
-  dependsOn ":beam-runners-google-cloud-dataflow-java-legacy-worker:shadowJar"
+    group = "Verification"
```

Comment:
I think the indention should be 2 here (though not all the groovy code adheres to this).

Reply:
Thanks. Fixed.